info.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "sOCUtilities",
+    "version": "1.0.0",
+    "fsrMinCompatibility": "7.4.0",
+    "type": "solutionpack",
+    "local": true,
+    "label": "SOC Utilities",
+    "dependencies": [],
+    "prerequisite": "",
+    "publisher": "Fortinet CSE",
+    "description": "This Solution pack is designed around utilities to aid in the use of SOAR",
+    "help": null,
```

Comment:
We need documentation created for this SP and its respective readme.md path should be mentioned here as well

Reply:
e.g. "help": "https://github.com/fortinet-fortisoar/solution-pack-typosquat-alert-investigation/blob/develop/README.md",

Reply:
Documentation can be created by following steps mentioned here: https://github.com/fortinet-fortisoar/how-tos/blob/main/documenting/README.md

Reply:
Added the documentation in Readme, contents, setup, and usuage md files.